### PR TITLE
This task completes Task 5 - GET /api/articles

### PR DESCRIPTION
**Pull request by yaiqbal**

### CODE REVIEW

File: __tests__/index.test.js [hunk 1]
```diff
@@ -83,4 +83,27 @@ describe('/api/articles/:article_id', () => {
...
+          expect(typeof article.article_id).toBe('number');
+          expect(typeof article.title).toBe('string');
+          expect(typeof article.topic).toBe('string');
+          expect(typeof article.author).toBe('string');
+          expect(typeof article.created_at).toBe('string');
+          expect(typeof article.votes).toBe('number');
+          expect(typeof article.article_img_url).toBe('string'); 
+          expect(typeof parseInt(article.comment_count)).toBe('number');
+          expect(article).not.toHaveProperty('body'); 
+        });
```
Comment (MKD01) [inline __tests__/index.test.js:106]: Really good test. I would recommend looking into .toMatchObject as a nice alternative.

**Review by MKD01 — COMMENTED** [review]
Really good work.

**Comment by yaiqbal** [comment]
Thank you MKD for the review comments.

File: __tests__/index.test.js (hunk 1 above)
Comment (yaiqbal) [reply __tests__/index.test.js:106]: Thank you. I have implemented .toMatchObject. This change will be part of the commit for Task 7.

**Pull request merged**